packages/template-retail-react-app/app/components/order-summary/index.jsx

```diff
@@ -18,11 +18,31 @@ import CartItemVariantName from '../item-variant/item-name'
 import CartItemVariantAttributes from '../item-variant/item-attributes'
 import CartItemVariantPrice from '../item-variant/item-price'
 import PromoPopover from '../promo-popover'
+import {useProducts} from 'commerce-sdk-react-preview'
 
 const CartItems = ({basket}) => {
-    const {totalItems, products} = useCurrentBasket({
-        shouldFetchProductDetail: true
-    })
+    const {totalItems} = useCurrentBasket()
```

Comment:
here we have 2 sources for basket data.

1. we pass basket as a prop
2. we call `useCurrentBasket` to get derived basket data

I'm sure this works, but i feel like this might makes the code more prone to bugs.

```suggestion
    const totalItems = basket?.productItems?.reduce((acc, item) => acc + item.quantity, 0) || 0
```